doc/book/app_server/luajit_getmetrics.rst

```diff
@@ -0,0 +1,449 @@
+    | gc_tabnum            | number of allocated table objects                |            |
+    +----------------------+--------------------------------------------------+------------+
+    | gc_total             | number of bytes of currently allocated memory    |            |
+    |                      | (normally equals gc_allocated minus gc_freed)    |            |
+    +----------------------+--------------------------------------------------+------------+
+    | gc_udatanum          | number of allocated udata objects                |            |
+    +----------------------+--------------------------------------------------+------------+
+    | jit_mcode_size       | total size of all allocated machine code areas   |            |      
+    +----------------------+--------------------------------------------------+------------+
+    | jit_snap_restore     | Overall number of snap restores (amount of guard | yes        |
+    |                      | assertions leading to stopping trace executions) |            |
+    |                      | See external `Snap tutorial`_                    |            |
+    +----------------------+--------------------------------------------------+------------+
+    | jit_trace_abort      | overall number of aborted traces                 | yes        |
+    +----------------------+--------------------------------------------------+------------+
+    | jit_trace_num        | number of JIT traces                             |            |
+    +----------------------+--------------------------------------------------+------------+
+    | strhash_hit          | number of strings being interned because, if a   | yes        |
+    |                      | string with the same value is found via the      |            |
+    |                      | hash, a new one is not created / allocated       |            |
+    +----------------------+--------------------------------------------------+------------+
+    | strhash_miss         | total number of strings allocations during       | yes        |
+    |                      | the platform lifetime                            |            |
+    +----------------------+--------------------------------------------------+------------+
+
+.. comment: Links are not inline because they would make the table cells wider.
+
+.. _Sweep phase description: http://wiki.luajit.org/New-Garbage-Collector#sweep-phase
+.. _Snap tutorial: https://ujit.readthedocs.io/en/latest/public/tut-snap.html
+
+Note: Although value names are similar to value names in
+`ujit.getmetrics() <https://ujit.readthedocs.io/en/latest/public/ujit-024.html#ujit-getmetrics>`_
+the values are not the same, primarily because many ujit numbers are not monotonic.
+
+Note: Although value names are similar to value names in :ref:`LuaJIT metrics <metrics-luajit>`,
+and the values are exactly the same, misc.getmetrics() is slightly easier
+because there is no need to ‘require’ the misc module.
+
```

Comment:
```suggestion
..  note::
    Although value names are similar to value names in
    `ujit.getmetrics() <https://ujit.readthedocs.io/en/latest/public/ujit-024.html#ujit-getmetrics>`_,
    the values are not the same,
    primarily because many ujit numbers are not monotonic.

..  note::

    Although value names are similar to value names
    in :ref:`LuaJIT metrics <metrics-luajit>`,
    and the values are exactly the same, misc.getmetrics() is slightly easier
    because there is no need to ‘require’ the misc module.

```

The second link <metrics-luajit> remains unparsed. Is it broken? Where should it lead? @pgulutzan 

I also recommend using "note" admonitions. They look nice, and they're in [our style guide](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/admonitions/).

Summoning @NickVolynkin just in case.

Reply:
The second link, i.e. :ref:`LuaJIT metrics <metrics-luajit>`,
should lead to metrics-luajit in
doc/book/monitoring$/metrics_reference.rst
When I clone, though I often forget, I include these steps:
git submodule init
git submodule update
mkdir -p doc/book/monitoring
cp -r modules/metrics/doc/monitoring/ doc/book
Do you include the same steps?

Reply:
I see. The correct link is now `<metrics-reference-luajit>`. I changed it to align it with the style guide
See line 369 here: https://github.com/tarantool/metrics/edit/master/doc/monitoring/metrics_reference.rst

Reply:
OK, thanks.